pyspecProcScripts/phasing.py

```diff
@@ -329,7 +328,7 @@ def hermitian_function_test(
         mask = mask.astype(float)
         norm = np.floor(mid_idx - B)
         norm = norm.astype(float)
-        mask /= (norm) * 2
+        mask /= ((norm) * 2)**2
```

Comment:
this is a another change that I just made pretty empirically, but it does seem to prevent the low residual for the low choice of echo center -- roughly here, the argument is that it's harder to match more points, and we seemed to be insufficiently accounting for that.  This one I'm really not sure about but worth @aabeaton testing on real data?